XRTK-Core/Packages/com.xrtk.core/Services/BoundarySystem/MixedRealityBoundarySystem.cs

```diff
@@ -663,7 +663,7 @@ public bool TryGetRectangularBoundsParams(out Vector2 center, out float angle, o
             }
 
             // Handle the user teleporting (boundary moves with them).
-            var transformedCenter = MixedRealityToolkit.CameraSystem.CameraRig.PlayspaceTransform.TransformPoint(
+            var transformedCenter = MixedRealityToolkit.CameraSystem.MainCameraRig.PlayspaceTransform.TransformPoint(
```

Comment:
We should check if the camera system is null here and fall back to `CameraCache.Main.parent`